app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -210,7 +210,7 @@ def update
     authorize @consumer_role, :update?
     save_and_exit = params['exit_after_method'] == 'true'
     mec_check(@person.hbx_id) if EnrollRegistry.feature_enabled?(:mec_check) && @person.send(:mec_check_eligible?)
-    @shop_coverage_result = EnrollRegistry.feature_enabled?(:shop_coverage_check) ? check_shop_coverage.success? : nil
+    @shop_coverage_result = EnrollRegistry.feature_enabled?(:shop_coverage_check) ? check_shop_coverage.success? && check_shop_coverage.success.present? : nil
```

Comment:
can you enclose this in brackets (check_shop_coverage.success? && check_shop_coverage.success.present?)